api/features_api_test.py

```diff
@@ -465,7 +465,10 @@ def test_patch__stage_changes(self):
       'stages': [
         {
           'id': self.ship_stage_1_id,
-          'intent_thread_url': new_intent_url,
+          'intent_thread_url': {
+            'field_display_name': 'shipped_milestone',
```

Comment:
Normally I think of a "display name" as something that is shown to users, but `shipped_milestone` is in snake-case so it looks more like a "field name" that still needs to be converted to a display name at some point.

Reply:
That's true. This property is misnamed, and is meant to represent the form field name on the client side (e.g. `shipped_milestone`) rather than the field in the database (`desktop_first`). It is used for writing activities and notification emails. I've changed the name to `form_field_name`. Hopefully this is better, but I am willing to change it if it's still unclear. 🙂